cpp/src/cider/exec/nextgen/context/RuntimeContext.cpp

```diff
@@ -22,6 +22,7 @@
 #include "exec/nextgen/context/RuntimeContext.h"
 #include "exec/module/batch/CiderArrowBufferHolder.h"
 #include "exec/nextgen/operators/extractor/AggExtractorBuilder.h"
+#include "tests/utils/ArrowArrayBuilder.h"
```

Comment:
Possible to move this `ArrowArrayBuilder.h ` out of  `test_util`? It doesn't make sense for nextgen to depend on test package. I think you can put it under `cider/util`.